terraform/shared/Makefile

```diff
@@ -31,10 +31,14 @@ plan: validate
 rename_resources: config
 	python $(project_root)/scripts/rename_resources.py
 
-.PHONY: apply
-apply: validate rename_resources
+.PHONY: terraform
+terraform: validate rename_resources
 	terraform apply
 
+.PHONY: deploy
```

Comment:
I prefer that there is correspondence between the target name and the 1st `terraform` argument.

Since the post_deploy script is fast we can just invoke it directly after `terraform apply`